config/operator/global/operator.template.yaml

```diff
@@ -41,8 +41,17 @@ spec:
             cpu: 100m
             memory: 20Mi
         ports:
-        - containerPort: 9876
+        - containerPort: 9443
           name: webhook-server
           protocol: TCP
+        volumeMounts:
+          - mountPath: /tmp/k8s-webhook-server/serving-certs
+            name: cert
+            readOnly: true
       terminationGracePeriodSeconds: 10
+      volumes:
```

Comment:
This is maybe overthinking it. I wonder if we want the secret to be in this file, because right now this yaml on its own will fail to deploy because the secret doesn't exist. You would need to deploy the webhook yaml also.

That might be a usability thing we can punt on in this PR though. I think right now most people will have the ability to deploy everything, and those that don't have the ability to edit yaml files. We might want to adjust in the future but this could work for now.